<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>188</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tri Color Light Simulator</string>
  </property>
  <widget class="QPushButton" name="bStartStop">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>StartStop</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbRed">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Red</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbOrange">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>92</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Orange</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbGreen">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>92</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Green</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
